### ISSUE TITLE
recv: failed (Unknown error) messages in rest wai example

### ISSUE THREAD

**Issue by Heather** [silkapp/rest#150]
When I run rest wai example

generate some client and then use `_run "localhost" (User.list [])` it works and I can see full info loaded by example but in rest-wai console I'm getting messages: `recv: failed (Unknown error)`

**Comment by hesselink** [comment]
What is the `_run` function? And how are you running the server (i.e. what command to start it)?

I just tried and can't reproduce here on OS X.

**Comment by Heather** [comment]
@hesselink I've pushed example on github to reproduce https://github.com/Heather/NeverLand/blob/master/client/src/APITest.hs#L12 it's really minimal based on rest-example

**Comment by hesselink** [comment]
Thanks, that's really useful! I've built the main package with `-fwai`, built the client, and ran both executables. This printed the following in the client:

```
PluginInfo {name = "plugin 1"}
PluginInfo {name = "plugin 2"}
PluginInfo {name = "plugin 3"}
```

In the server it printed

```
Starting warp server on http://localhost:3000
```

And nothing else. So I'm not seeing the messages you're seeing. It could be a windows specific issue, I guess. Sadly I don't know much about windows.

**Comment by Heather** [comment]
@hesselink yes, it works the same for me but after `Starting warp server on http://localhost:3000` line I saw `recv: failed (Unknown error)` message, I think I will invest some more time to get more related information and will post here as soon as I will find something related.
